src/index.scss

```diff
@@ -1,10 +1,16 @@
+@import 'carbon-components/scss/globals/scss/styles.scss';
 /* Remove overrides once Carbon bugs are fixed upstream. */
 
 /*
 The React <Content /> component uses the `main` element which IE11 doesn't recognize
 
 @link https://github.com/carbon-design-system/carbon/issues/2789
 */
+
 main {
   display: block;
 }
+
```

Comment:
The feature-flags mapping should come before the style import. See https://www.carbondesignsystem.com/tutorial/react/step-1/#add-ui-shell-sass